doc/examples/fonts.py

```diff
@@ -1,19 +1,20 @@
+#!/usr/bin/env python
```

Comment:
I am not a linux expert, but would it make sense to specify `python3` here instead of python?

Reply:
All of these scripts were written to be python2 compatible, that's why it's python and not python3.

Reply:
Originally these scripts supported python 2 and 3,
but in the 2.0 version support for python 2 will be dropped.

Reply:
I'd be happy to update the PR if you'd like, but currently setup.py reflects support for 2.7, I can update that in the PR too if you'd like

Reply:
If there are more places, let's save that for a new PR,
so that this doesn't become one of those "change everything" PR's.